### ISSUE TITLE
Questions about training

### ISSUE THREAD

**Issue by XiaozhenQiao** [Junyi42/LoGeR#16]
This is very interesting work, but I encountered a problem when reproducing the training on PI3: the results are similar to PI3 during full sequence training, but once I enable chunking, the model's performance drops significantly, and I don't know why.

**Comment by panxkun** [comment]
<img width="1420" height="697" alt="Image" src="https://github.com/user-attachments/assets/c4583eb0-300e-40f2-be83-1aa94806ad4a" />

I tried training the first stage based on the pretrained Pi3 model, but from the loss curve it seems that the loss can no longer keep decreasing. Should I continue waiting for it to go down, or does this indicate that there might be some issue?
